airflow-core/src/airflow/ui/public/i18n/locales/el/dags.json

```
@@ -0,0 +1,97 @@
+{
+  "assetSchedule": "{{count}} από {{total}} οντότητες ενημερώθηκαν",
+  "dagActions": {
+    "delete": {
+      "button": "Διαγραφή Dag",
+      "warning": "Αυτό θα αφαιρέσει όλα τα μεταδεδομένα που σχετίζονται με το Dag, συμπεριλαμβανομένων των εκτελέσεων και των εργασιών."
+    }
+  },
+  "favoriteDag": "Αγαπημένο Dag",
+  "filters": {
+    "allRunTypes": "Όλοι οι τύποι Εκτελέσεων",
+    "allStates": "Όλες οι καταστάσεις",
+    "favorite": {
+      "all": "Όλα",
+      "favorite": "Αγαπημένα",
```

```suggestion
      "favorite": "Αγαπημένο",
```

Same for the next line.